Extensions/tweaks.js

```diff
@@ -305,6 +305,25 @@ XKit.extensions.tweaks = new Object({
 		XKit.extensions.tweaks.process_wrap_tags_one_line();
 	}),
 
+	cpanel: function($panel) {
+		$panel
+		.find(".xkit-extension-setting-separator")
+			.css("text-transform", "none")
+			.css("margin-top", 0)
+			.css("padding", "15px")
+			.each(function() {
+				this.outerHTML = this.outerHTML.replace(/div/g, "summary");
+			});
+
+		$panel
+		.html($panel.html()
+			.replace(/<summary/g, "</details><details><summary")
+			.replace("</details>", "")
+			+ "</details>")
```

Comment:
ooooof. not sure i like munging the HTML with regex's so thoroughly :/ 

it's also relying on assumptions about where xkit_preferences is attaching its click handlers, which probably aren't always good assumptions. Can we go through and do this properly with DOM modification instead of html string modification? if not, it might just be cleaner to put this into xkit preferences.

Reply:
agreed that it'd be much better baked into xkit preferences. honestly i'm not too much of a fan of this PR myself code-wise either so i'll implement the feature in another PR i think